develop/devguide/Connector/ConnectionsSnmpRetrievingTables.md

```diff
@@ -34,204 +42,403 @@ Multiple methods are available for retrieving tables via SNMP in a protocol. The
+        <ColumnOption idx="1" pid="1002" type="snmp" options="" />
+        ...
+    </ArrayOptions>
+    <SNMP>
+        <Enabled>true</Enabled>
+        <OID type="complete" options=";multipleGetBulk:3"></OID>
+    </SNMP>
+</Param>
+<Param id="1001" trending="false">
+    <Name>InterfacesIndex</Name>
+    <Description>Instance [IDX] (Interfaces)</Description>
+    <Type>read</Type>
+    <Interprete>
+        <RawType>other</RawType>
+        <Type>string</Type>
+        <LengthType>next param</LengthType>
+    </Interprete>
+    <SNMP>
+        <Enabled>true</Enabled>
+        <OID type="complete">1.3.6.1.2.1.2.2.1.1</OID>
+        <Type>Integer32</Type>
+    </SNMP>
+</Param>
+<Param id="1002" trending="false">
+    <Name>InterfacesName</Name>
+    <Description>Name (Interfaces)</Description>
+    <Type>read</Type>
+    <Interprete>
+        <RawType>other</RawType>
+        <Type>string</Type>
+        <LengthType>next param</LengthType>
+    </Interprete>
+    <SNMP>
+        <Enabled>true</Enabled>
+        <OID type="complete">1.3.6.1.2.1.2.2.1.2</OID>
+        <Type>octetstring</Type>
+    </SNMP>
+</Param>
+```
 
 > [!NOTE]
 >
-> - The default number of rows that is retrieved in one request is 10.
-> - The number of rows to be retrieved in one request should be carefully chosen in order to reduce the risk of having an SNMP response that will be larger than the Path MTU of the path across the network traversed by the message. Messages with a large size are subject to fragmentation. Fragmentation is generally considered to be harmful, because it leads among others to a decrease in the reliability of the transfer of the messages. For a UDP packet inside a typical Ethernet frame, this typically means that the entire SNMP response should preferably stay below 1472 bytes (+8 bytes UDP header +20 bytes IPv4 header = 1500 bytes Ethernet payload). See also: [RFC 3416, chapter 2.3 Message Sizes](https://www.rfc-editor.org/rfc/rfc3416.html#section-2.3).
-> - The get bulk request was introduced in SNMPv2. Therefore, this method cannot be used for devices that only support SNMPv1.
+> - Choose the number of rows per request carefully to avoid SNMP responses that exceed the network Path MTU. Large responses may be fragmented, which can reduce reliability. For typical Ethernet networks, keep SNMP responses under 1472 bytes (plus 8 bytes for the UDP header and 20 bytes for the IPv4 header, totaling 1500 bytes). For more details, see [RFC 3416, section 2.3](https://www.rfc-editor.org/rfc/rfc3416.html#section-2.3).
+> - The GetBulk request is only available in SNMPv2 and later. This method is not supported on SNMPv1 devices.
 
-Capture observation:
+SNMP Communication Flow:
 
 - Get bulk requests are performed until all data is retrieved.
 - The max-repetitions field indicates how many rows will be retrieved per request. In DataMiner, you can define this field by specifying a number after the "multipleGetBulk" option. E.g multipleGetBulk:5.  
```

Comment:
Further explain here how it actually works?
similar to MultipleGetNext where the first request will do a GetNext on each column but in a bulk manner (request more than 1 next OID for each column), then starting again from the last retrieved cell OID for each column.